content/spin/dynamic-configuration.md

```diff
@@ -141,7 +141,7 @@ You can supply runtime configuration by providing a value for the `--runtime-con
 
 ### Key Value Store Runtime Configuration
 
-Spin provides built-in key-value storage. This storage is backed by an SQLite database embedded in Spin by default. However, the Spin runtime configuration file (runtime-config.toml) can be updated to not only modify the SQLite configuration but also choose to use a different backing store. The available store options are the embedded SQLite database, an external Redis database or Azure CosmosDB.
+Spin provides built-in key-value storage. This storage is backed by an SQLite Database embedded in Spin by default. However, the Spin runtime configuration file (runtime-config.toml) can be updated to not only modify the SQLite configuration but also choose to use a different backing store. The available store options are the embedded SQLite Database, an external Redis database or Azure CosmosDB.
```

Comment:
This is definitely _not_ the Cloud brand name, it's just an ordinary non-proper-named SQLite database on your local disk.

```suggestion
Spin provides built-in key-value storage. This storage is backed by an SQLite database embedded in Spin by default. However, the Spin runtime configuration file (runtime-config.toml) can be updated to not only modify the SQLite configuration but also choose to use a different backing store. The available store options are the embedded SQLite database, an external Redis database or Azure CosmosDB.
```